src/kv/map_handle.h

```diff
@@ -88,6 +88,26 @@ namespace kv
       return read_handle.has(KSerialiser::to_serialised(key));
     }
 
+    /** Get version when this key was last written to, by a previous
```

Comment:
I think `kv/api.rst` will need to include `get_previous_version` as a member of `ReadableMapHandle` to show up in the docs.